src/common.build.pre.props

```diff
@@ -131,6 +132,7 @@
     <Link>
       <EnableCOMDATFolding>true</EnableCOMDATFolding>
       <OptimizeReferences>true</OptimizeReferences>
+      <AdditionalDependencies>d3dcompiler.lib</AdditionalDependencies>

```

Comment:
Sorry- bit confused. I think we already _have_ an `AdditionalDependencies` in one of our common files. It might be `common.build.post.props`, though, to enforce a linking order.

Can you move this there, and ensure that it comes _after_ `onecoreuap_apiset`? Thanks/sorry!

Reply:
(this one is still here)